src/node_crypto.cc

```diff
@@ -544,14 +545,18 @@ void SecureContext::Initialize(Environment* env, Local<Object> target) {
   t->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "kTicketKeyIVIndex"),
          Integer::NewFromUnsigned(env->isolate(), kTicketKeyIVIndex));
 
-  t->PrototypeTemplate()->SetAccessor(
+  Local<FunctionTemplate> ctx_getter_templ =
+    FunctionTemplate::New(env->isolate(),
+                          CtxGetter,
+                          Local<Value>(),
```

Comment:
We usually use `env->as_external()` for the parameter right here, so that we can get access to the `Environment` object in the callback itself. Again, here and below.

Reply:
You mean instead of Local<Value>()?

Reply:
Yes.

Reply:
Tests completed fine with just `Local<Value>()`, so that's a bit worrying, but it's perhaps out of the scope of this PR to add tests for that. I've now switched to `env->as_external` wherever it was previously used.